### ISSUE TITLE
Meteor 1.3 - Handlebars.templates is sometimes undefined

### ISSUE THREAD

**Issue by maxenceC** [cmather/meteor-handlebars-server#35]
Whenever my local server start, or when I deploy to my servers with mupx, Handlebars.templates will sometimes be undefined, resutling in errors.

I didn't have this problem on Meteor 1.2, neither on the previous versions.

It happens 1 time over 3 build

**Comment by fakenickels** [comment]
Yes, the Handlebars namespace doesn't get properly populated. At my company we rewrote the package with the new Meteor plugin API and solved all these problems.
Try out our version on https://atmospherejs.com/astrocoders/handlebars-server

**Comment by sferoze** [comment]
@grsabreu awesome thanks!
